tg_fm_news.py

```diff
@@ -26,8 +36,22 @@ def print_current_news(interval):
             item_text = item.text
             item_href = "https://sputnik.by/geoBelarus" + item.get("href")
             all_news_dict[item_text] = item_href  # Запись значений в словарь
-            bot.send_message(chat_id, f"{item_text}: {item_href}")
-        # bot.send_message(chat_id, f"{item_text}: {item_href}")
+            # print(f"{item_text}: {item_href}")
+            data = item_text
+            cursor.execute("SELECT * FROM news WHERE heading = %s", (data,))
```

Comment:
* Use URL instead of heading as unique key
* Use 'SELECT EXISTS ...' query

Reply:
Use URL instead of heading as unique key - fixed